jsonschema-core.md

```diff
@@ -523,11 +523,23 @@ to inform the user of the evaluation path that produced the error or annotation.
 
 ### Keyword Interactions
 
-Keyword behavior MAY be defined in terms of the annotation results of
-[subschemas](#root) and/or adjacent keywords (keywords within the same schema
-object) and their subschemas. Such keywords MUST NOT result in a circular
-dependency. Keywords MAY modify their behavior based on the presence or absence
-of another keyword in the same [schema object](#schema-document).
+Keywords MAY modify their behavior based on the presence or absence of another
```

Comment:
It's not just the "presence" of another keyword, it's also the "value" of that keyword that can modify behavior.

Reply:
I feel that may be implicit or understood.  I'm not sure why a keyword would be sensitive to the presence of another keyword but not its value.

Reply:
I think it's implicit enough for informal discussion, but not for a specification. I think the spec should be explicit.